avocado/core/status/repo.py

```diff
@@ -71,3 +92,8 @@ def process_raw_message(self, raw_message):
     @property
     def result_stats(self):
         return {key: len(value) for key, value in self._by_result.items()}
+
+    def get_task_status(self, task_id):
+        if task_id not in self._status:
+            return None
```

Comment:
There is any reason to have this if, when you have the default values for the .get() bellow? Seems to me that only the return line would be enough here.

Reply:
You're right, dropping this.